crt_portal/cts_forms/templates/forms/complaint_view/index/filter-controls.html

```diff
@@ -9,7 +9,7 @@ <h2 class="intake-section-title">Filters</h2>
   <div class="usa-form intake-filters intake-checkbox">
     {% include 'forms/complaint_view/index/filters/checkbox_status_filter.html' %}
   </div>
-  <div class="intake-filters">
+  <div class="usa-form intake-filters">
```

Comment:
Note: this is necessary for our faux checkboxes to show up (.crt-checkbox__label::before) -- I checked and this doesn't visually impact the filters but it'd be nice if the reviewer double-checked as well.